pages/about/index.tsx

```diff
@@ -32,6 +33,54 @@ interface TeamSection {
   members: string;
 }
 
+type ArticleType = {
+  title: string;
+  url: string;
+  sourceName: string;
+  lang: string | null; // TODO Language type
+  publishedDate: string;
+};
+
+type ArticlesListProps = {
+  articles: ArticleType[];
+};
+
+type ArticleProps = {
+  article: ArticleType;
+};
+
+const Article = ({ article }: ArticleProps): JSX.Element => {
+  const handleArticleClick = (e: React.MouseEvent<HTMLElement, MouseEvent>, url: string) => {
+    e.stopPropagation();
+    url && window.open(url);
+  };
+  return (
+    <div
+      className="w-full lg:max-w-full lg:flex cursor-pointer group px-2 py-1 my-1 rounded shadow border"
+      onClick={(e) => handleArticleClick(e, article.url)}
+    >
+      <h3 className="group-hover:text-primary-blue">{article.title}</h3>
+      <div className="font-light">
+        <span className="inline-block lg:pl-2 pr-2">{article.sourceName}</span>
+        <span className="inline-block pr-2">{new Date(article.publishedDate).toLocaleDateString()}</span>
+        {article.lang && <span className="inline-block">{article.lang}</span>}
```

Comment:
I think instead of rendering the language code `ru` we might want to either: 
* Translate the code to a localized word: "Rusky"
* Or better:  render a small flag emoji instead 🇷🇺 

It's more visually interesting and you don't need 20 extra translations. 

You would have a map of flag emojis 

```ts
const flagEmojis = {
 ru: "🇷🇺",
 uk: "🇺🇦",
 etc... 
}
``` 

And then just render `flagEmojis[article.lang]`

@met What do you think? :)

Reply:
I considered emojis too, thank you)

Reply:
@m5rk1t @JuroUhlar I like emoji ideas too, but careful on Windows emoji do not show flag, only letter "CZ", other apps like Twitter/Facebook make some hacks (using own flag images for that).
But that is a cosmetic problem, that could be fixed later.